include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -4769,12 +4778,18 @@
 
   ORT_API2_STATUS(OrtGraph_MaxNodeIndex, const OrtGraphViewer* graph, _Out_ int* out);
 
+  // ORT_API2_STATUS(OrtGraph_CreateModel, const OrtGraphViewer* graph, _Outptr_ OrtModel** out);
+
+  ORT_API2_STATUS(OrtGraph_GetOutputs, const OrtGraphViewer* graph, _Out_ size_t* len, _Outptr_ const OrtNodeArg*** args);
```

Comment:
Do not expose NodeArg, Get what you want inside a function and make it transparent to the caller. See OrtNode_GetIthOutputName as an example

Or you can refer the PR [https://github.com/microsoft/onnxruntime/pull/18090](https://github.com/microsoft/onnxruntime/pull/18090), see the file include/onnxruntime/interface/graph/graph.h, about what functions are exposed to traverse the graph